homeassistant/components/unifiprotect/camera.py

```diff
@@ -40,7 +40,11 @@ def get_camera_channels(
 
         is_default = True
         for channel in camera.channels:
-            if channel.is_rtsp_enabled:
+            # G4 Doorbell Pro as a 4th camera channel for package camera,
+            # it is _always_ named "Package Camera"
+            if channel.name == "Package Camera":
```

Comment:
Maybe make this a constant from `pyunifiprotect`?

Reply:
As I was thinking about it more, I was thinking of adding a property `is_package_channel` or something. So, we do not need to check the FPS either. Just consistent way of checking.

Reply:
> As I was thinking about it more, I was thinking of adding a property `is_package_channel` or something. So, we do not need to check the FPS either. Just consistent way of checking.

That would be even better 👍